panels/source/arrangers/OtherArrangers.js

```diff
@@ -45,9 +45,12 @@ enyo.kind({
 			var f = this.container.toIndex;
 			var c$ = this.getOrderedControls(f);
 			var o = Math.floor(c$.length/2);
+		        var wf = (s === 0 && f == (c$.length-1));
```

Comment:
Yep - I was wondering if it should instead be:
```
var wb = (s === 0 && f == (c$.length-1)); // we're moving backward, from the first panel to the last panel
var wf = (s == (c$.length-1) && f === 0); // we're moving forward, from the last panel to the first panel
```
Maybe our definitions of wrapping forward/backward are different. :)